usr/etc/flatpak/user/install

```diff
@@ -1,22 +0,0 @@
-org.mozilla.firefox
```

Comment:
Maybe a stupid question, but will this lose an existing users' configuration?  Like bookmarks, themes and whatever else you can store in some of these apps?

Reply:
We definitely need to test this, but it should be kept since the system and user Firefox both use the same directory for their configuration and we don't specify the option to delete app data to the remove command.